src/javascripts/ng-admin/Crud/button/maBackButton.js

```diff
@@ -7,16 +7,19 @@ define(function () {
         return {
             restrict: 'E',
             scope: {
-                'size': '@'
+                'size': '@',
+                'label': '@',
             },
             link: function ($scope) {
+                $scope.label = $scope.label || 'Back';
+
                 $scope.back = function () {
                     $window.history.back();
                 };
             },
             template:
 '<a class="btn btn-default" ng-class="size ? \'btn-\' + size : \'\'" ng-click="back()">' +
-    '<span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>&nbsp;Back' + 
+    '<span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>&nbsp;{{ label }}' +
```

Comment:
Use `{{ ::label }}` to reduce overhead ([one-time binding](https://docs.angularjs.org/guide/expression#one-time-binding)). Same for each directive in the PR.